### PR TITLE
Sophia Bell airport_challenge pull request

### PR DESCRIPTION
**Pull request by Kittaru87**

### CODE REVIEW

**Comment by srikaanthtb** [comment]
code is easy to read and things are clearly defined, easy to understand, plane.rb could possibly be refactored, it's not immediately obvious what weather.rb is trying to do. Adding a comment will help.

File: lib/plane.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require_relative 'airport'
+
+class Plane
+    
+  attr_reader :location, :status
+
+  def initialize
```
Comment (srikaanthtb) [inline lib/plane.rb]: file could be refactored into just the initialise method

File: lib/weather.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require_relative 'airport'
+
+class Weather
+
+  def stormy?
```
Comment (srikaanthtb) [inline lib/weather.rb]: not immediately obvious what this does

File: spec/plane_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require 'plane'
```
Comment (srikaanthtb) [inline spec/plane_spec.rb]: easy to read with well defined tests

File: lib/weather.rb (hunk 2 above)
Comment (Kittaru87) [reply lib/weather.rb]: now changed

**Comment by srikaanthtb** [comment]
lol i completely forgot about this, doing some long overdue github cleanup are we?

**Pull request closed without merging**